src/telemetry.ts

```diff
@@ -1625,39 +1702,51 @@ export class IEventNamePropertyMapping {
         feature: ['KernelPicker'],
         properties: commonClassificationForResourceSpecificTelemetryProperties
     };
+    /**
+     * Time take for Jupyter session to be idle.
+     */
     [Telemetry.SessionIdleTimeout]: TelemetryEventInfo<never | undefined> = {
         owner: 'donjayamanne',
         feature: 'N/A',
         source: 'N/A',
         tags: ['KernelStartup']
     };
+    /**
+     * Sent when we display a message informing the user about Jupyter not being installed (or not detected).
+     */
     [Telemetry.JupyterNotInstalledErrorShown]: TelemetryEventInfo<never | undefined> = {
         owner: 'donjayamanne',
         feature: 'N/A',
         source: 'N/A',
         tags: ['KernelStartup']
     };
+    /**
+     * Sent when user installs Jupyter.
+     */
     [Telemetry.UserInstalledJupyter]: TelemetryEventInfo<never | undefined> = {
         owner: 'donjayamanne',
         feature: 'N/A',
         source: 'N/A',
         tags: ['KernelStartup']
     };
-    /*
+    /**
```

Comment:
`/*` is not a valid jsDoc comment, it needs to be `/**`
This is how the typescript compiler looks for comments, hence its not a requirement of the tool, but a requirement of typescript to detect comments.